x-pack/legacy/plugins/xpack_main/index.js

```diff
@@ -52,6 +52,13 @@ export const xpackMain = (kibana) => {
             defaultMessage:
               'Recipient email address for X-Pack admin operations, such as Cluster Alert email notifications from Monitoring.',
           }),
+          deprecation: {
+            message: i18n.translate('xpack.main.uiSettings.adminEmailDeprecation', {
+              defaultMessage:
+                'This setting is deprecated and will not be supported in Kibana 7.0. Please configure `xpack.monitoring.cluster_alerts.email_notifications.email_address` in your kibana.yml settings.',
```

Comment:
Let's use `monitoring.cluster_alerts.email_notifications.email_address` instead. The `xpack.` one is deprecated